cwms_radar_api/src/main/java/cwms/radar/ApiServlet.java

```diff
@@ -103,20 +119,21 @@ public void init() throws ServletException{
         om.setPropertyNamingStrategy(PropertyNamingStrategies.KEBAB_CASE);
         om.registerModule(new JavaTimeModule());            // Needed in Java 8 to properly format java.time classes
 
+        AccessManager accessManager = buildAccessManager();
+
+
         javalin = Javalin.createStandalone(config -> {
             config.defaultContentType = "application/json";
             config.contextPath = context;
             config.registerPlugin(new OpenApiPlugin(getOpenApiOptions()));
             config.enableDevLogging();
             config.requestLogger( (ctx,ms) -> logger.finest(ctx.toString()));
-            //config.addStaticFiles("/static",Location.CLASSPATH);
```

Comment:
This should stay. This is how the index page gets served... unless that got replaced; going in order.

Reply:
nevermind, we comment that out weeks ago.... this is why we should just delete things, like you have.